.github/workflows/ms.analysisservices.servers.yml

```diff
@@ -69,6 +79,7 @@ jobs:
   job_module_pester_validation:
```

Comment:
@AlexanderSehr would it make sense to change the job name to match other names that use `staticValidation`. E.g ==> `job_module_staticValidation`  or `job_module_static_validation` ?

Reply:
Sure, why not. I guess the original reason was to imply that this runs Pester - but I don't think that's too important on that level. Will update

Reply:
Done. I went with `job_module_static_validation`